app.js

```diff
@@ -238,6 +238,11 @@ const ws = new WorkflowStep('superbot_help_request', {
             ]
         })
 
+        if (!jiraId) {
+            console.log("Posting help request to Jira Failed.");
```

Comment:
not terribly useful as you don't have the error message but i guess worth a try to see if this is the code path it's hitting